### DISCUSSION TITLE
Unable to render page 404 on css and js

### DISCUSSION THREAD

**Discussion by thewhaleiam** [awcodes/filament-tiptap-editor#405] [Q&A] [answered]
Hello, I have attempted to install and followed the instructions however I keep coming across the same issue:

When accessing via the browser application is attempting to retrieve and produce the following errors

**css/awcodes/tiptap-editor/tiptap.css?v=3.2.35.0 net::ERR_ABORTED 404 (Not Found)**
 **js/awcodes/tiptap-editor/components/tiptap.js?v=3.2.35.0 net::ERR_ABORTED 404 (Not Found)**. 

My files look like:

resource -> css -> filament -> admin -> theme.css

```
@import '../../../../vendor/filament/filament/resources/css/theme.css';

@import '../../../../vendor/awcodes/filament-tiptap-editor/resources/css/plugin.css';

@config './tailwind.config.js';
```

tailwind.config.js file 

```import colors from 'tailwindcss/colors'
import forms from '@tailwindcss/forms'
import typography from '@tailwindcss/typography'

export default {
    content: [
        './resources/**/*.blade.php',
        './vendor/filament/**/*.blade.php',
        './vendor/awcodes/filament-tiptap-editor/resources/**/*.blade.php',
    ],
    darkMode: 'class',
    theme: {
        extend: {
            colors: {
                danger: colors.rose,
                primary: colors.blue,
                success: colors.green,
                warning: colors.yellow,
            },
        },
    },
    plugins: [
        forms,
        typography,
    ],
}
```

and the vite.config.js file is as follows:

```import { defineConfig } from 'vite';
import laravel from 'laravel-vite-plugin';
export default defineConfig({
    plugins: [
        laravel({
            input: [
                'resources/css/filament/admin/theme.css',
                'resources/css/app.css',
                'resources/js/app.js',
            ],
            refresh: true,
        }),
    ],
});
```

and the postcss.config.js is as follows:

```module.exports = {
    plugins: {
        'tailwindcss/nesting': {},
        tailwindcss: {},
        autoprefixer: {},
    },
}
```

This is how the page is rendered: 
<img width="1497" alt="Screenshot 2024-06-12 at 15 51 41" src="https://github.com/awcodes/filament-tiptap-editor/assets/139074731/e65d779d-83b5-4964-8abf-5af474afcae9">

In the build folder there is nothing that pertains to awcodes, as it is imported on the theme.css I would expect that the css is handled by the minified file and not to be loaded as above.

I have tried rolling back to older versions (3.0, 3.1 and 3.2), if I use version 3.3 nothing renders to the page. Any pointers or advice would be greatly appreciated.

**Comment by awcodes** [comment, marked as answer]
Try running `php artisan filament:assets`

**Reply by thewhaleiam** [reply]
Thanks, that worked

**Comment by broqit** [comment]
<img width="728" alt="Знімок екрана 2024-07-13 о 23 40 19" src="https://github.com/user-attachments/assets/cd9dd426-ad20-4459-884d-aaec14ddd922">
Hello.
I hame a same problem but css file is ok.

**Reply by broqit** [reply]
Sorry. The problem was because of the ->tools([]) method that was added by default. Maybe it should be removed from the code example in the documentation.

**Reply by awcodes** [reply]
The code example in the readme is meant to show what is possible. Not a cut and paste. 😀
